apps/site/src/lib/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,47 @@
+import hack from "@/assets/logos/hack.svg";
+import mail from "@/assets/logos/mail.svg";
+import facebook from "@/assets/logos/facebook.svg";
+import instagram from "@/assets/logos/instagram.svg";
+import linkedin from "@/assets/logos/linkedin.svg";
+import youtube from "@/assets/logos/youtube.svg";
+import Image from "next/image";
+
+import styles from "./Footer.module.scss";
+
+const Footer = () => {
+
+	const imageArray = [
+	[hack, "Hack Logo"],
+	[mail, "Email"],
+	[facebook, "Facebook"],
+	[instagram, "Instagram"],
+	[linkedin, "Linkedin"],
+	[youtube, "Youtube"]]
```

Comment:
Suggestion: It would be more readable if you used a list of hashmaps/objects here instead of nested arrays for enhanced readability. See [`Footer.tsx`](https://github.com/HackAtUCI/zothacks-site-2023/blob/main/apps/site/src/components/Footer/Footer.tsx) for an example. Additionally, we usually place constants like these outside of the component.